src/components/gui/gui.jsx

```diff
@@ -49,9 +49,11 @@ const GUIComponent = props => {
                     <Box
                         width={250}
                     >
-                        <TabSelected>
+                        <Tab
+                            selected={true}
```

Comment:
Great! Now to make tab styles rely on the state, you just need to store some property on the state of the GUI that says which tab is selected.

```jsx
const selected = this.state.selected

// ..other tag things..

<Tab selected={selected === 'scripts'}>
  <TabText>Scripts</TabText>
</Tab>

<Tab selected={selected === 'costumes'}>
  <TabText>Costumes</TabText>
</Tab>

<Tab selected={selected === 'sounds'}>
  <TabText>Sounds</TabText>
</Tab>
```